metaflow/plugins/conda/conda_step_decorator.py

```diff
@@ -79,8 +84,11 @@ def _lib_deps(self):
         base_deps = self.base_attributes['libraries']
         deps.update(base_deps)
         step_deps = self.attributes['libraries']
-        if isinstance(step_deps, collections.Mapping):
-            deps.update(step_deps)
+        if isinstance(step_deps, (unicode, basestring)):
+            step_deps = step_deps.strip('"{}"')
```

Comment:
Maybe to be safe, consider the strip `'"{}\''` (the second " is irrelevant in any case).

Reply:
Done. Thanks for noticing the typo!